### PR TITLE
Blogpost - Pupil LSL Plugin

### PR DESCRIPTION
**Pull request by nathakits**
Copytext needs checking.

### CODE REVIEW

**Comment by nathakits** [comment]
I was thinking we could include an image to briefly explain and show how it works.

File: contents/articles/2016-11_pupil-lsl-plugin/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+date: Fri Nov 18 2016 13:37:50 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Introducing Pupil LSL relay plugin that relays pupil data, gaze data and notifications to the lab streaming layer..."
+featured_img: ""
+featured_img_thumb: "" 
+---
+
+The [Lab Streaming Layer](https://github.com/sccn/labstreaminglayer) (LSL) is a system that provides unified collection of measurement time series between programs, computers and devices over a network for distributed signal transport, time synchronization and data collection. LSL has an extensive range of supported measurement modalities including eye tracking. 
+
+Introducing [Pupil LSL relay plugin](https://github.com/sccn/labstreaminglayer/tree/master/Apps/PupilLabs). A [Pupil Capture](https://github.com/pupil-labs/pupil/wiki/Pupil-Capture) plugin that relays pupil data, gaze data and notifications to the Lab Streaming Layer which users can link the data to other inlets in the network.
```
Comment (willpatera) [inline contents/articles/2016-11_pupil-lsl-plugin/index.md]: Move this to first paragraph.

Change first sentence to:
"We are excited to introduce the [Pupil + Lab Streaming Layer relay plugin]...

File: contents/articles/2016-11_pupil-lsl-plugin/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+--- 
+title: Pupil plugin for Lab Streaming Layer
+date: Fri Nov 18 2016 13:37:50 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Introducing Pupil LSL relay plugin that relays pupil data, gaze data and notifications to the lab streaming layer..."
+featured_img: ""
+featured_img_thumb: "" 
+---
+
+The [Lab Streaming Layer](https://github.com/sccn/labstreaminglayer) (LSL) is a system that provides unified collection of measurement time series between programs, computers and devices over a network for distributed signal transport, time synchronization and data collection. LSL has an extensive range of supported measurement modalities including eye tracking.
```
Comment (willpatera) [inline contents/articles/2016-11_pupil-lsl-plugin/index.md]: Move to second paragraph.

File: contents/articles/2016-11_pupil-lsl-plugin/index.md (hunk 1 above)
Comment (willpatera) [inline contents/articles/2016-11_pupil-lsl-plugin/index.md]: The plugin works with Pupil Capture to relay pupil data, gaze data, and notifications to LSL. Users can link the data to other inlets in the network.

**Pull request merged**